tendrl/commons/flows/import_cluster/__init__.py

```diff
@@ -6,4 +6,6 @@ def __init__(self, *args, **kwargs):
         super(ImportCluster, self).__init__(*args, **kwargs)
 
     def run(self):
+        self.parameters['Node.fqdn'] = NS.node_context.fqdn
+        self.parameters['Service.name'] = 'collectd'
```

Comment:
https://github.com/Tendrl/commons/pull/661/files/d4b7b0d652528b213a427db53b043652408c5950#diff-1a5320aa0fd1eea178e6ed255509c9c4L298  due to https://github.com/Tendrl/commons/pull/661/files/d4b7b0d652528b213a427db53b043652408c5950#diff-1a5320aa0fd1eea178e6ed255509c9c4R94